src/sage/combinat/root_system/root_lattice_realizations.py

```diff
@@ -1229,7 +1231,7 @@ def negative_roots(self):
             """
             if not self.cartan_type().is_finite():
                 raise ValueError("%s is not a finite Cartan type" % self.cartan_type())
-            return self.positive_roots().map(attrcall('__neg__'))
+            return self.positive_roots().map(attrcall('__neg__'), is_injective=True)
```

Comment:
why this change?  `is_injective` is the default, and also what I'd assume that `map` does.